pkg/pillar/zedcloud/deferred.go

```diff
@@ -17,15 +17,14 @@ import (
 )
 
 // Example usage:
-// deferredChan := zedcloud.InitDeferred(zedcloudCtx)
+// ctx := zedcloud.CreateDeferredCtx(zedcloudCtx, ...)
 // select {
-//      case change := <- deferredChan:
-//		zedcloud.HandleDeferred(zedcloudCtx, change)
+//      case change := <- ctx.Ticker.C:
+//          ctx.HandleDeferred(...)
```

Comment:
Nit but the change aka event argument is still there in HandleDeferred.

Reply:
I put '...' to avoid repeating all the arguments. Seems there is no need in such an example to expose exact arguments we have for this function.